github/ci/prow/files/jobs/kubevirt/kubevirt-presubmits.yaml

```diff
@@ -391,7 +391,7 @@ presubmits:
       fork-per-release: "true"
       k8s.v1.cni.cncf.io/networks: multus-cni-ns/sriov-passthrough-cni
     always_run: true
```

Comment:
That should have been set to `false` too, I think.

Reply:
I thought so too, but Felix wanted it to run on every PR so we can track the progress of the fix.
I don't mind changing it...

Reply:
`required` is meant for the branch-protector to mark the job as required (if I can for instance just hit the merge button when jobs are failing). But tide will simply not merge anything when it sees a failing job.

Reply:
ah, I had the wrong assumption that it will allow us to merge with this job failing :/ anyway, the issue should be solved now when the faulty node is offline. Do you want me to still set this job to not always run?